home-manager/ssh.nix

```diff
@@ -46,16 +46,17 @@ in
     controlMaster = "auto";
     controlPersist = "10m";
 
-    addKeysToAgent = "yes";
-
     # Enable custom or temporary config without `home-manager switch`
     includes = [
       "${sshDir}/config.local"
     ];
 
     # https://www.clear-code.com/blog/2023/4/3/recommended-ssh-config.html
     # https://gitlab.com/clear-code/ssh.d/-/blob/main/global.conf?ref_type=heads
+    # @TODO: Update AddKeysToAgent since release-24.05. See #394
```

Comment:
💭 If I completed the task https://github.com/kachick/dotfiles/issues/243, this task should be simple...